### PR TITLE
Initial openapi.yml and package.json (no endpoints yet)

### PR DESCRIPTION
**Pull request by fboucquez**
Endpoints and schemas would be added one by one, not in this PR. We should create a list of endpoints/issues from https://nemproject.github.io/ and work on them, most likely a project board . Schemas should be generated from catbuffer when possible.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "nem-openapi",
+  "version": "0.0.1",
+  "description": "OpenAPI Specification of Nis1 Rest interface",
```
Comment (gimre-xymcity) [inline package.json:4]: Nem/NEM REST interface

**Review by gimre-xymcity — APPROVED** [review]

File: spec/openapi.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+externalDocs:
+    description: 'NEM Developers'
+    url: 'https://nemproject.github.io/'
+servers:
+    - url: 'http://localhost:7890'
+      description: 'Local development'
+    - url: 'http://hugetestalice2.nem.ninja:7890'
+      description: 'Huge Alice Testnet'
+    - url: 'http://hugealice2.nem.ninja:7890'
+      description: 'Huge Alice Mainnet'
```
Comment (Jaguar0625) [inline spec/openapi.yml:17]: you should use NGL (@wayonb) servers

**Review by Jaguar0625 — CHANGES_REQUESTED** [review]
have you reviewed these?
https://gist.github.com/kengos/3b0620636fc6775b07f3e4b1288b5c3b

**Comment by fboucquez** [comment]
> have you reviewed these?
> https://gist.github.com/kengos/3b0620636fc6775b07f3e4b1288b5c3b

I didn't know it. Thanks! What should we do? I think we still need an official nem open API. Splitting the file is also recommended. 

We can use that file as a point of reference while adding the other endpoints. Each endpoint should be tested against a running server though.

File: spec/openapi.yml (hunk 2 above)
Comment (fboucquez) [reply spec/openapi.yml:17]: I thought Huge Alices were the (most) official nodes. Happy to change the URLs.

Comment (Jaguar0625) [reply spec/openapi.yml:17]: yes, please change. ask @wayonb for nodes.

**Review by Jaguar0625 — APPROVED** [review]

File: spec/openapi.yml (hunk 2 above)
Comment (fboucquez) [reply spec/openapi.yml]: I've asked, http://54.170.97.221:7890 is the only NGL mainnet. No domain yet.

**Pull request merged**